src/classes/anAddress.cls

```diff
@@ -0,0 +1,27 @@
+public interface anAddress {
```

Comment:
I was originally thinking we might have two implementations - one for our regular address and one for seasonal. But it seems like we might just have one concrete class for NPSP_Address.  In that case this interface can be removed, but curious to hear others thoughts.

Reply:
I don't think we'll need this. Seasonal vs. non-Seasonal Addresses are all still addresses. The only difference is the start/end periods, which are technically still attributes of an address even if they don't have a value.